src/lib/actions/send-task-failure.js

```diff
@@ -0,0 +1,52 @@
+const { errors, status } = require('../../constants');
+
+function sendTaskFailure(params, activities) {
+  /* check request parameters */
+  if (params.cause &&
+    (typeof params.cause !== 'string'
+    || params.cause.length > 32768)
+  ) {
+    throw new Error(`${errors.common.INVALID_PARAMETER_VALUE}: --cause`);
+  }
+  if (params.error &&
+    (typeof params.error !== 'string'
+    || params.error.length > 256)
+  ) {
+    throw new Error(`${errors.common.INVALID_PARAMETER_VALUE}: --error`);
+  }
+  if (typeof params.taskToken !== 'string'
+    || params.taskToken.length < 1
+    || params.taskToken.length > 1024
+  ) {
+    // NOTE: Could also be INVALID_TOKEN
```

Comment:
we should also check what is AWS behaviour

Reply:
Will check

Reply:
Checked. It returns InvalidToken.
-> Will update the code